test/e2e/e2e_test.go

```diff
@@ -133,6 +146,67 @@ var _ = Describe("controller", Ordered, func() {
 				return nil
 			}
 			EventuallyWithOffset(1, verifyOneRunningPod, time.Minute, time.Second).Should(Succeed())
+
+			By("validating that the nginx pod is running as expected")
+			//var nodeName string
+			verifyOneRunningNginxPod := func() error {
```

Comment:
time to share with verifyOneRunningPod ? Take a label selector and number of pods expected?